### PR TITLE
fix: update additional completions for help, almost-all, dereference

### LINKED ISSUE #511: fix: update additional completions for --almost-all, --dereference, --help
This issue tracks missing updates to completions started in #503. I have a pull request coming through shortly.

### PR DESCRIPTION
**Pull request by cgorski**
Contains additional completions missed in #503.

Resolves #511

### CODE REVIEW

**Comment by cgorski** [comment]
If the domain experts for these shells could scrutinize my work, it would be much appreciated, as I don't have extensive shell scripting experience.

File: completions/fish/eza.fish [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Meta-stuff
 complete -c eza -s v -l version -d "Show version of eza"
-complete -c eza -s '?' -l help -d "Show list of command-line options"
+complete -c eza -l help -d "Show list of command-line options"
```
Comment (MartinFillon) [inline completions/fish/eza.fish:3]: just FYI `-h` will be assigned to help when #197 will be introduced as it is an industry standard

**Review by MartinFillon — APPROVED** [review]

File: completions/fish/eza.fish (hunk 1 above)
Comment (cgorski) [reply completions/fish/eza.fish:3]: Oops - I missed that. I’ll make a correction.

**Review by cafkafk — APPROVED** [review]
I think these are okay, but I'm also not an expert in neither zsh nor bash

**Pull request merged**